doc/classes/Object.xml

```diff
@@ -971,6 +983,15 @@
 				[b]Note:[/b] In C#, [param property] must be in snake_case when referring to built-in Godot properties. Prefer using the names exposed in the [code]PropertyName[/code] class to avoid allocating a new [StringName] on each call.
 			</description>
 		</method>
+		<method name="set_edited">
+			<return type="void" />
+			<param index="0" name="edited" type="bool" />
+			<description>
+				Set whether the object has been edited based on the [param edited] value.
+				[b]Note:[/b] This is primarily used by the editor for [Resource] based objects to track their modified state. For example, changing a scene, a resource in the inspector, or a script will call this method with [code]true[/code]. Saving the scene, script, or resource reset the edited state by calling this method with [code]false[/code].
```

Comment:
The "for example" wording needs some work I think, it's unclear what it means, I'd say "changing a scene" here sounds like switching tabs, maybe something like:
```suggestion
				[b]Note:[/b] This is primarily used by the editor for [Resource] based objects to track their modified state. For example, any changes to an open scene, a resource in the inspector, or an edited script will cause this method to be called with [code]true[/code]. Saving the scene, script, or resource resets the edited state by calling this method with [code]false[/code].
```